README.md

```diff
@@ -136,18 +147,24 @@ These types are unsupported as function parameter types or return types for func
 * [`void`](https://www.typescriptlang.org/docs/handbook/2/functions.html#void), [`object`](https://www.typescriptlang.org/docs/handbook/2/functions.html#object), [`unknown`](https://www.typescriptlang.org/docs/handbook/2/functions.html#unknown), [`never`](https://www.typescriptlang.org/docs/handbook/2/functions.html#never), [`any`](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#any) types - to accept and return arbitrary JSON, use `sdk.JSONValue` instead
 * `null` and `undefined` - unless used in a union with a single other type
 
-
-### Impure/pure functions
-If you write a function that performs a read-only operation, you should mark it with the `@readonly` JSDoc tag, and it will be exposed as an NDC function, which will ultimately show up as a GraphQL query field in Hasura.
-
-```typescript
-/** @readonly */
-export function add(x: number, y: number): number {
-  return x + y;
+### Parallel execution
+If functions are involved remote relationships in your Hasura metadata, then they may be queried in a [batch-based fashion](https://hasura.github.io/ndc-spec/specification/queries/variables.html). In this situation, any async functions that are marked with the `@readonly` JSDoc tag may be executed in parallel. The default degree of parallelism per query request to the connector is 10, but you may customise this by using the `@paralleldegree` JSDoc tag on your function.
+
+``` typescript
+/**
+ * This function will only run up to 5 http requests in parallel per query
+ *
+ * @readonly
+ * @paralleldegree 5
```

Comment:
Could you use params to an annotation such as `@parallel(degree: 5)`? No worries if not.

Reply:
Is that a thing in JSDoc? I've never seen that syntax before... what's something that does that already that I can use for reference?

Reply:
I guess it's not a thing. There is the `@ann {bla}` convention, but we can just keep it as is.

Reply:
That's too bad, I liked your syntax 😟